sdk/src/index.d.ts

```diff
@@ -17,11 +17,44 @@ import * as BigNum from '@liskhq/bignum';
 import * as cryptography from '@liskhq/lisk-cryptography';
 import * as transactions from '@liskhq/lisk-transactions';
 
-declare const Application: any;
+type AnyClass = { new (): any };
+type TransactionClass = AnyClass;
+type ModuleClass = AnyClass;
+
+declare class Application {
+	readonly logger: any;
+	readonly config: any;
+
+	constructor(genesisBlock: any, config?: any);
+	run(): Promise<void>;
+	registerTransaction(
+		transactionClass: TransactionClass,
```

Comment:
I got `Argument of type 'typeof TransferTransaction' is not assignable to parameter of type 'AnyClass` :cry:

Reply:
Will check now

Reply:
@shuse2 Core is not complaining to me :(. Can you still reproduce this issue?

Reply:
yes, i just tried, but it seems like it is still happening.
```
src/index.ts:47:26 - error TS2345: Argument of type 'typeof TransferTransaction' is not assignable to parameter of type 'AnyClass'.

47  app.registerTransaction(TransferTransaction, {
                            ~~~~~~~~~~~~~~~~~~~

src/index.ts:51:26 - error TS2345: Argument of type 'typeof SecondSignatureTransaction' is not assignable to parameter of type 'AnyClass'.

51  app.registerTransaction(SecondSignatureTransaction, {
                            ~~~~~~~~~~~~~~~~~~~~~~~~~~

src/index.ts:55:26 - error TS2345: Argument of type 'typeof DelegateTransaction' is not assignable to parameter of type 'AnyClass'.

55  app.registerTransaction(DelegateTransaction, {
                            ~~~~~~~~~~~~~~~~~~~

src/index.ts:59:26 - error TS2345: Argument of type 'typeof VoteTransaction' is not assignable to parameter of type 'AnyClass'.

59  app.registerTransaction(VoteTransaction, {
                            ~~~~~~~~~~~~~~~

src/index.ts:63:26 - error TS2345: Argument of type 'typeof MultisignatureTransaction' is not assignable to parameter of type 'AnyClass'.

63  app.registerTransaction(MultisignatureTransaction, {
                            ~~~~~~~~~~~~~~~~~~~~~~~~~

src/index.ts:67:26 - error TS2345: Argument of type 'typeof DappTransaction' is not assignable to parameter of type 'AnyClass'.

67  app.registerTransaction(DappTransaction, {
                            ~~~~~~~~~~~~~~~

src/index.ts:71:26 - error TS2345: Argument of type 'typeof InTransferTransaction' is not assignable to parameter of type 'AnyClass'.

71  app.registerTransaction(InTransferTransaction, {
                            ~~~~~~~~~~~~~~~~~~~~~

src/index.ts:75:26 - error TS2345: Argument of type 'typeof OutTransferTransaction' is not assignable to parameter of type 'AnyClass'.

75  app.registerTransaction(OutTransferTransaction, {
```
18 errors like this

Reply:
@shuse2 just fixed it thanks, should work now